packages/asset-swapper/src/utils/market_operation_utils/types.ts

```diff
@@ -50,7 +51,7 @@ export interface NativeFillData extends FillData {
 }
 
 export interface RfqtFillData extends FillData {
-    quote: RfqtIndicativeQuoteResponse;
+    quote: IndicativeQuote;
```

Comment:
Why IndicativeQuote instead of RFQTIndicativeQuote?

Reply:
This code is actually unused, so I removed it in https://github.com/0xProject/0x-monorepo/pull/2582/commits/187dd2fdc3aab3901084466e47c0975e0331e426